### DISCUSSION TITLE
ORCID und GND-ID beim BibTex-Import für Personen

### DISCUSSION THREAD

**Discussion by vgerlach** [OPUS4/community#13] [Ideas]
Es gibt den Wunsch für Personen auch die ORCID und/oder GND-ID beim Import mit anzugeben. In der Import-Rule Person.php werden derzeit 2 Trenner berücksichtigt: **,** für die Trennung von Vor- und Nachname und **and** für die Trennung verschiedener Personen.
Ein möglicher Vorschlag mit dem neuen Trenner **:** könnte so aussehen:
author = {ORCID:0000-0003-4037-1597:GND:110269500:Hermanussen,Michael and ORCID:0000-0003-1954-7578:GND:1163672890:Scheffler, Christiane}

Eine etwas verallgemeinerte Form mit Nutzung eines weiteren zusätzlichen Trenners **+** könnte wie folgt aussehen:
author = {ORCID:0000-0003-4037-1597+GND:110269500+Hermanussen,Michael and GND:1163672890+ORCID:0000-0003-1954-7578+Scheffler, Christiane and ORCID:0000-0003-1954-7578+Scheffler, Christiane+GND:1163672890}

**Comment by starbuck242** [comment]
Ich finde den Vorschlag, ein zusätzliches bzw. anderes Zeichen zur Abtrennung von Gruppen, z.B. das "+" zu verwenden sehr gut. Macht es übersichtlicher für die menschliche Wahrnehmung und wenn es auch technisch sauberer zu trennen ist, wäre das aus meiner Sicht eine gute Lösung und den Import ein gutes Stück mächtiger.

**Comment by j3nsch** [comment]
Die erste Variante ist sicherlich machbar, aber da wird `:` für die Trennung zwischen den einzelne Informationseinheiten, ORCID, GND, Name verwendet und zwischen dem "FeldNamen", z.B. ORCID und dem Wert. Das macht das Parsing unhandlicher. Die Trennung mit `+` ist einfacher. Es muss darauf geachtet werden, dass die Einträge in unterschiedlicher Reihenfolge auftauchen können, wie im zweiten Beispiel.

**Comment by j3nsch** [comment]
Die Funktionalität wurde mit OPUS4/opus4-bibtex#69 umgesetzt und wird spätestens mit OPUS 4.8.1 veröffentlicht.
